runtime/moonriver/src/lib.rs

```diff
@@ -1419,6 +1419,18 @@ impl pallet_multisig::Config for Runtime {
 	type WeightInfo = moonbeam_weights::pallet_multisig::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const MaxStorageRoots: u32 = 10;
+}
+
+impl pallet_relay_storage_roots::Config for Runtime {
+	type MaxStorageRoots = MaxStorageRoots;
+	type RelaychainStateProvider = cumulus_pallet_parachain_system::RelaychainDataProvider<Self>;
+	// TODOs update weights
+	//type WeightInfo = moonbeam_weights::pallet_relay_storage_roots::WeightInfo<Runtime>;
+	type WeightInfo = ();
+}
+
```

Comment:
De changes should only be included in moonbase, right?

Reply:
The precompile will only be included in Moonbase. I believe we can still include this pallet in Moonriver and Moonbeam, as it does not depend on the precompile. What are your thoughts, @librelois?